core/tests/Test.Microsoft.Identity.LabInfrastructure/ILabService.cs

```diff
@@ -31,6 +31,6 @@ namespace Test.Microsoft.Identity.LabInfrastructure
 {
     public interface ILabService
     {
-        IUser GetUser(UserQueryParameters query);
+        ILabResponse GetUser(UserQueryParameters query);
```

Comment:
>GetUser [](start = 21, length = 7)

GetLabResponseFromUser ... or we need to find something better.... it's not just getting the user now.